src/webview/components/options-widget.tsx

```diff
@@ -399,7 +397,11 @@ export class OptionsWidget extends React.Component<OptionsWidgetProps, OptionsWi
                 this.props.updateRenderOptions({ wordsPerGroup: Number(value) });
                 break;
             case InputId.GroupsPerRow:
-                this.props.updateRenderOptions({ groupsPerRow: Number(value) });
+                if (isNumber(value)) {
+                    this.props.updateRenderOptions({ groupsPerRow: Number(value) });
+                } else {
+                    this.props.updateRenderOptions({ groupsPerRow: value });
+                }
```

Comment:
See comments above re: the `isNumber` function. This code is fairly redundant: if something is a number, we don't need make it a number, and we can use it directly; if it isn't a number, we do use it directly anyway.